css-contain-2/Overview.bs

```diff
@@ -2280,6 +2303,7 @@ This appendix is <em>informative</em>.
 Changes from <a href="https://www.w3.org/TR/2022/WD-css-contain-2-20220917/">2022-09-17 Working Draft</a>
 </h3>
 
+* defined [=proximity to the viewport=] and used it in the clarification 4.
```

Comment:
What's "the clarification 4"?

Reply:
There's a section called "Restrictions and Clarifications" which has some items listed, I'll update this to link to it.